### PR TITLE
Add dependabot config

### PR DESCRIPTION
**Pull request by stuggi**
Use dependabot to keep our go.mod files up to date and in sync

### CODE REVIEW

**Review by karelyatin — APPROVED** [review]
LGTM, IIUC after this merges daily we may have PRs like https://github.com/openstack-k8s-operators/keystone-operator/pull/123, right?

**Comment by stuggi** [comment]
> LGTM, IIUC after this merges daily we may have PRs like [openstack-k8s-operators/keystone-operator#123](https://github.com/openstack-k8s-operators/keystone-operator/pull/123), right?

yes, correct

**Pull request merged**
